l10n/en/products/vpn/platforms/shared.ftl

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/desktop/
+
+# Shared
+vpn-cta-headline = Let's get started
+vpn-shared-cta = See pricing & availability
+vpn-shared-day-guarantee = 30-day money-back guarantee
+vpn-shared-what-youll-get = What you’ll get with { -brand-name-mozilla-vpn }:
+
+vpn-shared-feature-item-protection = Protection for up to { -vpn-available-devices } devices
+vpn-shared-feature-item-access = Access to servers in { -vpn-available-countries }+ countries
+vpn-shared-feature-item-encryption = Device-level encryption
+vpn-shared-feature-item-bandwidth = No bandwidth restrictions
+vpn-shared-feature-item-logs = No online activity logs now or ever
+
+-vpn-available-devices = 5
+-vpn-available-countries = 280
+-vpn-features-servers = { -vpn-available-devices }+ servers in more than { -vpn-available-countries }+ countries
```

Comment:
I would probably try avoid mixing sentances like this using placeholders, as it may lead to odd results in some languages. Keeping sentances together will probably be easier for translators.

Reply:
Should these be variables instead too?

Reply:
The number of devices and countries should be variables yes, but you're also using part of a sentance here like a placeholder that gets inserted into another sentance? It seems like an efficient thing to do at a code level, but I think there's a change it might be confusing for localizers, and potentially lead to translations that don't quite join together right.